src/services/ReportGenerationService.ts

```diff
@@ -62,11 +62,14 @@ class ReportGenerationService {
                 return template.workbook.xlsx.writeBuffer()
                 .then((buffer: Excel.Buffer) => {
                     return {
-                        fileName: `ATFReport_${moment(activity.startTime).tz(TimeZone.LONDON).format("DD-MM-YYYY")}_${moment(activity.startTime).tz(TimeZone.LONDON).format("HHmm")}_${activity.testStationPNumber}_${activity.testerName}.xlsx`,
-                        fileBuffer: buffer
+                        /* tslint:disable */fileName: `ATFReport_${moment(activity.startTime).tz(TimeZone.LONDON).format("DD-MM-YYYY")}_${moment(activity.startTime).tz(TimeZone.LONDON).format("HHmm")}_${activity.testStationPNumber}_${activity.testerName}.xlsx`,
```

Comment:
Need to re-enable linting after this block - tslint doesn't do single line exclusion for some reason:
https://palantir.github.io/tslint/usage/rule-flags/

Reply:
fixed